js/index.js

```diff
@@ -40,3 +40,52 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+let secondImg = document.getElementById("cta-img");
+secondImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+let thirdImg = document.getElementById("middle-img");
+thirdImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+let navItem = document.getElementsByTagName("a");
+console.log(navItem);
+
+let h1 = document.querySelector("h1");
+
+h1.innerHTML = "Dom <br> is <br> awesome";
+
+let btn = document.querySelector("button");
+btn.innerText = "Get Started";
+
+let navItems = document.querySelectorAll('a');
+navItems[0].innerText ="Services";
+navItems[1].innerText ="Product";
+navItems[2].innerText ="Vision";
+navItems[3].innerText ="Features";
+navItems[4].innerText ="About";
+navItems[5].innerText ="Contact";
+
+let h4Els =    document.querySelectorAll('h4');
+console.log(h4Els);
+h4Els[0].innerText ="Features";
+h4Els[1].innerText = "About";
+h4Els[2].innerText = "Services";
+h4Els[3].innerText = "Product";
+h4Els[4].innerText = "Vision";
+
+let paragraphs = document.querySelectorAll('p');
+
+paragraphs[0].innerText ="Features content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
+paragraphs[1].innerText ="About content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
+paragraphs[2].innerText ="Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
+paragraphs[3].innerText ="Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
+paragraphs[4].innerText ="Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
+
+
+h4Els[5].innerText ="Contact";
+
+paragraphs[5].innerText ="123 Way 456 Street Somewhere, USA";
+paragraphs[6].innerText = "1 (888) 888-8888";
+paragraphs[7].innerHTML ="sales@greatidea.io";
+
+paragraphs[8].innerText ="Copyright Great Idea! 2018";
```

Comment:
This is all great. I would say that unless you'll actually use HTML elements, then innerText would be better. innerHTML has some downsides that we didn't get into.